src/index.css

```diff
@@ -18,4 +18,8 @@
   .content-max-width {
     max-width: 960px;
   }
+
+  .content-min-height {

```

Comment:
why index.css? Should we put it in organization module?

Reply:
I believe we should actually use this in every page though to make a minimum content height (from navbar to footer). But the value of height can be change based on UX/UI.